### ISSUE TITLE
FormHelper::year not displaying correctly the empty message

### ISSUE THREAD

**Issue by andrecavallari** [cakephp/cakephp#5818]
http://book.cakephp.org/3.0/en/views/helpers/form.html#creating-year-inputs
According to the docs, if I use a string as empty option it will display the message instead of an empty option, but the following code outputs an empty field, with no text.
```php
echo $this->Form->year('purchased', ['empty'=>'Year of purchase']);
```
It outputs the select with the year, but no message "Year of purchase"

**Comment by dereuromark** [comment]
Can you also bin the resulting HTML code please?

**Comment by andrecavallari** [comment]
Ok...
https://gist.github.com/andrecavallari/a80e56a745ec52f2a24c
As simple as that...

**Comment by markstory** [comment]
Fixed in #5821
